File: server/PredlaganjeSaradnjeIRCDemo/PredlaganjeSaradnjeIRC.Data/Model/Location.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text;

namespace PredlaganjeSaradnjeIRC.Data.Model
{
    public class Location
    {
        [Required]
        public int Id { get; set; }
        public virtual City City { get; set; }
        [Required]
        public string StreetName { get; set; }
        [Required]
        public int Number { get; set; }
        public int? Storey { get; set; }
        public int? Door { get; set; }
    }
}

File: client/fpis-app/src/model/Location.ts
import { City } from "./City";

export class Location{
    id:number;

    city:City;

    streetName:string;

    number:number;

    storey:number;

    door:number;

    constructor(id:number,city:City,streetName:string,number:number,storey:number,door:number){
        this.id=id;
        this.city=city;
        this.streetName=streetName;
        this.number=number;
        this.storey=storey;
        this.door=door;
    }
}